x-pack/solutions/observability/plugins/profiling/public/routing/index.tsx

```diff
@@ -232,7 +218,7 @@ const routes = {
                 element: (
                   <RouteBreadcrumb
                     title={i18n.translate('xpack.profiling.breadcrumb.topnFunctions', {
-                      defaultMessage: 'Top N',
+                      defaultMessage: 'Top N functions',
```

Comment:
NIT: I compared it with the title and there we have `TopN`, should we make it consistent? So use `Top N` there or use `TopN` (without space) here so they match? wdyt? 
<img width="770" alt="image" src="https://github.com/user-attachments/assets/26af10d8-7a83-4a79-8cbb-398d2a8371c8" />